lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -3170,19 +3169,32 @@ def model_add_material(model, material_name)
       end
     end
 
-    # OpenStudio::logFree(OpenStudio::Info, 'openstudio.standards.Model', "Adding material: #{material_name}")
-
     # Get the object data
-    data = model_find_object(standards_data['materials'], 'name' => material_name)
-    unless data
-      OpenStudio.logFree(OpenStudio::Warn, 'openstudio.standards.Model', "Cannot find data for material: #{material_name}, will not be created.")
-      return false
-      # @todo change to return empty optional material
+    if material_name.include?('Simple Glazing')
+      material_type = 'SimpleGlazing'
+      # default values
+      u_factor = 1.23
+      shgc = 0.61
+      vt = 0.81
+      material_name.split.each_with_index do |item, i|
+        if item == 'U'
```

Comment:
I brought up that point by email, I thought it wasn't too robust but you seem to be in favor of it. Now the way I see it is that we're setting up the naming convention to use so perhaps if we flag obvious errors it might be okay? For example if the name doesn't contain or includes a typo in `Simple Glazing` it just won't be found in the database so an error will be returned, we could add a warning when a property has not been found in the name (either because of a typo or actually missing) and let the user know that we're using a default value in place, and also when the same property is specified twice in the name (as per your example). Happy to discuss that further.

Reply:
Oops. yeah you did bring that up in October and I forgot.

I'm ok doing name first, but then keeping the database option in case someone wants to deviate from it. If we do name first, we need a bit more error checking. If we do the data first, that risks slowing down code for the majority of users, but perhaps captures likely instances where the name differs. I'm unsure of which to choose. I guess name first with good error checking?

Reply:
That sounds good to me, I'll push some changes this week to address this. Thanks!